DOCS-1290: Document what data actions owners and operators can take (#2133)
viamrobotics/docs: docs/manage/fleet/organizations.md

You can also use organizations for departments or other entities that can have one or more [locations](/manage/fleet/locations/).
If you are familiar with Google Drive, you can think of an organization as a shared drive.

> {{<imgproc src="/manage/rbac.png" resize="700x" declaredimensions=true alt="Organization page">}}

Reviewer: ```suggestion
{{<imgproc src="/manage/rbac.png" resize="600x" declaredimensions=true alt="Organization page">}}
```

> {{<imgproc src="/manage/rbac.png" resize="600x" declaredimensions=true alt="Organization page">}}

Reviewer: ```suggestion
{{<imgproc src="/manage/rbac.png" resize="900x" declaredimensions=true alt="Organization page">}}
```

### Invite users to your organization

To invite a user to your organization, click on the Org's **Settings** in the top right of the navigation bar.
In the members section of the page, click on **Grant access** and enter their email address.

> Then select the resource that you would like to grant the user access to well as the user's and click **Invite**.

Reviewer: ```suggestion
Then select the resource that you would like to grant the user access to and the designated role and click **Invite**.
```

> {{< imgproc alt="The user invitation menu on the Organization settings page." src="/manage/app-usage/invite-user.png" resize="800x" declaredimensions=true >}}

Reviewer: ```suggestion
{{< imgproc alt="The user invitation menu on the Organization settings page." src="/manage/app-usage/invite-user.png" resize="600x" declaredimensions=true >}}
```

> {{< imgproc alt="The user invitation menu on the Organization settings page." src="/manage/app-usage/invite-user.png" resize="600x" declaredimensions=true >}}

Reviewer: ```suggestion
{{< imgproc alt="The user invitation menu on the Organization settings page." src="/manage/app-usage/invite-user.png" resize="900x" declaredimensions=true >}}
```
